src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Event class contains the name, completion status
+ * and duration of a task.
+ */
+public class Event extends Task {
+    protected LocalDate duration;
```

Comment:
Variable name `duration` might come off as misleading. I would think that `duration` implies how long the `Event` would last rather than when the `Event` is.